src/profile-logic/process-profile.js

```diff
@@ -459,7 +462,7 @@ function _extractJsFunction(
   }
   const fileName = stringTable.indexForString(scriptURI);
   const lineNumber = parseInt(jsMatch[3], 10);
-
+  const columnNumber = parseInt(jsMatch[4], 10);
```

Comment:
Because the column number is optional, we can get a non-number value here (`parseInt(undefined)` is `NaN`, which is falsy). So you can write this like this:
```js
const columnNumber = jsMatch[4] ? parseInt(jsMatch[4], 10) : null;
```